mkdocs/docs/getting-started/quick-start.md

```diff
@@ -1163,6 +1543,50 @@ As the experiment progresses, you should eventually see that all of the objectiv
         }
         ]
         ```
+=== "Seldon"
+
+    ```shell
+    kubectl get vs routing-rule -o json --watch | jq .spec.http[0].route
+    ```
+
+    ??? info "Look inside traffic summary"
+        ```json
+        [
+          {
+            "destination": {
+              "host": "iris-default.ns-baseline.svc.cluster.local",
+              "port": {
+                "number": 8000
+              }
+            },
+            "headers": {
+              "response": {
+                "set": {
+                  "version": "iris-v1"
+                }
+              }
+            },
+            "weight": 25
+          },
+          {
+            "destination": {
+              "host": "iris-default.ns-candidate.svc.cluster.local",
+              "port": {
+                "number": 8000
+              }
+            },
+            "headers": {
+              "response": {
+                "set": {
+                  "version": "iris-v2"
+                }
+              }
+            },
+            "weight": 75
+          }
+	
+        ```    
+
 
 As the experiment progresses, you should see traffic progressively shift from the baseline version to the candidate version.
 
```

Comment:
> You created an Iter8 experiment with A/B testing pattern and progressive deployment pattern. In each iteration, Iter8 observed the latency and error-rate metrics collected by Prometheus, and the user-engagement metric from New Relic;

Could you please change New Relic to `New  Relic/Prometheus` above?

Reply:
yes